IfcDistributionSystem Rail Relocated Pset_EnvironmentalCondition (#312)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/r/RainIntensity_1HJs_aZFnBKP1bnlrIzkxt/Documentation.md

> Maximum level of rain intensity that the equipment complies with

Reviewer: ```suggestion
Maximum level of rain intensity that the equipment complies with.
```

> Maximum level of rain intensity that the equipment complies with

Reviewer 2: ```suggestion
Maximum level of rain intensity that the equipment can operate effectively in.
```
